File: Practica-java/src/_3_Estructuras_Condicionales/Ejercicio_5.java
package _3_Estructuras_Condicionales;
import java.util.Scanner;
public class Ejercicio_5 {
    public static void main(String[] arg){
        Scanner entrada = new Scanner(System.in);
        char letra;
        System.out.println("Ingrese un caracter: ");
        letra=entrada.next().charAt(0);

        switch(letra) {
            case 'a':
            case 'e':
            case 'i':
            case 'o':
            case 'u':System.out.println("Es una vocal minuscula"); break;
            default: System.out.println("No es una vocal minuscula"); break;
        }
    }
}

File: Practica-cpp/Trabajo_practico_1/3-Estructuras_Condicionales/Ejercicio_5.cpp
/*
 * 5. Escriba un programaque lea la entrada estandar de un caracter e indique en la salida esandar si el
 * caracter es una vocal minusculo o no
 */

#include <iostream>
using namespace std;
int main() {
    char letra;
    cout<<"Ingrese un caracter: ";cin>>letra;

    switch(letra) {
        case 'a':
        case 'e':
        case 'i':
        case 'o':
        case 'u':cout<<"Es una vocal minuscula"; break;
        default: cout<<"No es una vocal minuscula"; break;
    }

    return 0;
}

/*
 * 6. Escriba un programa que lea la entrada estandar un caracter e indique en la
 * salida estandar si el caracter es una vocal minuscula, es una vocal mayuscula
 * o no es una vocal
 */

